src/components/asthma/components/AsthmaAirQualities/AsthmaAirQualities.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import './AsthmaAirQualities.css';
+import { AsthmaAirQuality } from '../../model';
+import { useInitializeView } from '../../../../helpers/Initialization';
+import { asthmaDataService, getAsthmaDataStatusColor, getAsthmaDataStatusText } from '../../helpers';
+import { LoadingIndicator, SingleDataPoint, UnstyledButton } from '../../../presentational';
+import { AsthmaAirQualitiesPreviewState, previewData } from './AsthmaAirQualities.previewData';
+import MyDataHelps from '@careevolution/mydatahelps-js';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGear } from '@fortawesome/free-solid-svg-icons';
+import language from '../../../../helpers/language';
+
+export interface AsthmaAirQualitiesProps {
```

Comment:
Is this something that has to be coupled with Asthma? or can it be more general?

Reply:
We can probably create a general component for displaying AQI in a similar fashion, but I think this one is pretty asthma specific in that the data being loaded and the statuses being computed are dependent upon whether we are loading them for a particular date or to reflect the data being used to compute the Asthma control state.

The statuses and status colors are also Asthma specific at this time.